lib/jekyll-admin/server/page.rb

```diff
@@ -34,6 +39,18 @@ class Server < Sinatra::Base
 
       private
 
+      def html_content?
```

Comment:
Ahh, yes, because the constructor expects the file to exist. Good catch. What about creating a dummy class that includes `convertible`?